`rms_seed` required

**How to reproduce:**

1) Add _Simulation time series sensitivity_
2) Add tornado chart.
3) Attach the two using data channels.

This will give as error message in console:
```
view.tsx:107  Error: SensitivityResponseCalculator: Reference sensitivity rms_seed not found in ensemble
    at new ze (sensitivityResponseCalculator.ts:57:19)
    at Oe (view.tsx:104:51)
    at G1 (react-dom.production.min.js:167:137)
    at pg (react-dom.production.min.js:197:258)
    at JC (react-dom.production.min.js:292:88)
    at QC (react-dom.production.min.js:280:389)
    at m8 (react-dom.production.min.js:280:320)
    at qf (react-dom.production.min.js:280:180)
    at WC (react-dom.production.min.js:268:209)
    at k (scheduler.production.min.js:13:203)
```

**The user only see in frontend that there is no data provided through data channel:** 

![image](https://github.com/equinor/webviz/assets/31612826/7d7d40b8-55e1-41b6-b102-0de2b762943e)


**Possible solutions include:**

Instead of requiring `rms_seed`, let user choose reference seed in tornado chart module. Could let `*_seed` be default if it exists, like now.

--- a/frontend/src/modules/SimulationTimeSeriesSensitivity/settings.tsx
+++ b/frontend/src/modules/SimulationTimeSeriesSensitivity/settings.tsx
@@ -96,7 +96,7 @@ export function settings({ moduleContext, workbenchSession, workbenchServices }:
                 setSelectedSensitivities(sensitivityNames);
             }
         },
-        [sensitivityNames]
+        [computedEnsemble]
     );
     const sensitivityOptions: SelectOption[] = sensitivityNames.map((name) => ({
         value: name,

--- a/frontend/src/modules/TornadoChart/loadModule.tsx
+++ b/frontend/src/modules/TornadoChart/loadModule.tsx
@@ -6,6 +6,8 @@ import { view } from "./view";
 
 const defaultState: State = {
     plotType: PlotType.TORNADO,
+    referenceSensitivityName: null,
+    sensitivityNames: [],
     selectedSensitivity: null,
     responseChannelName: null,
 };

--- a/frontend/src/modules/TornadoChart/sensitivityResponseCalculator.ts
+++ b/frontend/src/modules/TornadoChart/sensitivityResponseCalculator.ts
@@ -49,11 +49,12 @@ export class SensitivityResponseCalculator {
     constructor(
         sensitivities: EnsembleSensitivities,
         ensembleResponse: EnsembleScalarResponse,
-        referenceSensitivity = "rms_seed"
+        referenceSensitivity: string | null
     ) {
         this._ensembleResponse = ensembleResponse;
         this._sensitivities = sensitivities;
-        if (!this._sensitivities.hasSensitivityName(referenceSensitivity)) {
+
+        if (!referenceSensitivity || !this._sensitivities.hasSensitivityName(referenceSensitivity)) {
             throw new Error(
                 `SensitivityResponseCalculator: Reference sensitivity ${referenceSensitivity} not found in ensemble`
             );

--- a/frontend/src/modules/TornadoChart/sensitivityTable.tsx
+++ b/frontend/src/modules/TornadoChart/sensitivityTable.tsx
@@ -60,7 +60,7 @@ const SensitivityTable: React.FC<SensitivityTableProps> = (props) => {
                 [TableColumns.TRUE_HIGH]: numFormat(sensitivityResponse.highCaseAverage),
                 [TableColumns.LOW_REALS]: sensitivityResponse.lowCaseRealizations.length,
                 [TableColumns.HIGH_REALS]: sensitivityResponse.highCaseRealizations.length,
-                [TableColumns.REFERENCE]: "rms_seed",
+                [TableColumns.REFERENCE]: props.sensitivityResponseDataset.referenceSensitivity,
             }));
         setTableRows(rows);
     }, [props.sensitivityResponseDataset, props.hideZeroY]);

--- a/frontend/src/modules/TornadoChart/settings.tsx
+++ b/frontend/src/modules/TornadoChart/settings.tsx
@@ -1,10 +1,32 @@
+import React from "react";
+
 import { ModuleFCProps } from "@framework/Module";
 import { ChannelSelect } from "@framework/components/ChannelSelect";
+import { Dropdown } from "@lib/components/Dropdown";
 import { Label } from "@lib/components/Label";
+import { R } from "@tanstack/react-query-devtools/build/legacy/devtools-c71c5f06";
 
 import { State } from "./state";
 
 export function settings({ moduleContext, workbenchServices }: ModuleFCProps<State>) {
+    const sensitivityNames = moduleContext.useStoreValue("sensitivityNames");
+    const [referenceSensitivityName, setReferenceSensitivityName] = React.useState<string | null>(null);
+    const setModuleReferenceSensitivityName = moduleContext.useSetStoreValue("referenceSensitivityName");
+    React.useEffect(
+        function propogateReferenceSensitivityName() {
+            setModuleReferenceSensitivityName(referenceSensitivityName);
+        },
+        [referenceSensitivityName]
+    );
+
+    if (!referenceSensitivityName && sensitivityNames.length > 0) {
+        if (sensitivityNames.includes("rms_seed")) {
+            setReferenceSensitivityName("rms_seed");
+        } else {
+            setReferenceSensitivityName(sensitivityNames[0]);
+        }
+    }
+
     return (
         <>
             <Label text="Data channel" key="data-channel-x-axis">
@@ -12,6 +34,13 @@ export function settings({ moduleContext, workbenchServices }: ModuleFCProps<Sta
                     moduleContext={moduleContext}
                     channelName="response"
                     broadcaster={workbenchServices.getBroadcaster()}
+                />
+            </Label>
+            <Label text="Reference sensitivity">
+                <Dropdown
+                    value={referenceSensitivityName ?? ""}
+                    options={sensitivityNames.map((s) => ({ label: s, value: s }))}
+                    onChange={setReferenceSensitivityName}
                 />
             </Label>
             ,

--- a/frontend/src/modules/TornadoChart/settings.tsx
+++ b/frontend/src/modules/TornadoChart/settings.tsx
@@ -4,7 +4,6 @@ import { ModuleFCProps } from "@framework/Module";
 import { ChannelSelect } from "@framework/components/ChannelSelect";
 import { Dropdown } from "@lib/components/Dropdown";
 import { Label } from "@lib/components/Label";
-import { R } from "@tanstack/react-query-devtools/build/legacy/devtools-c71c5f06";
 
 import { State } from "./state";
 

--- a/frontend/src/modules/TornadoChart/state.ts
+++ b/frontend/src/modules/TornadoChart/state.ts
@@ -10,6 +10,8 @@ export type SelectedSensitivity = {
 
 export interface State {
     plotType: PlotType;
+    referenceSensitivityName: string | null;
+    sensitivityNames: string[];
     selectedSensitivity: SelectedSensitivity | null;
     responseChannelName: string | null;
 }

--- a/frontend/src/modules/TornadoChart/view.tsx
+++ b/frontend/src/modules/TornadoChart/view.tsx
@@ -7,6 +7,8 @@ import { useEnsembleSet } from "@framework/WorkbenchSession";
 import { useElementSize } from "@lib/hooks/useElementSize";
 import { BarChart, TableChart, Tune } from "@mui/icons-material";
 
+import { isEqual } from "lodash";
+
 import SensitivityChart from "./sensitivityChart";
 import {
     EnsembleScalarResponse,
@@ -29,9 +31,10 @@ export const view = ({ moduleContext, workbenchSession, workbenchSettings, initi
     const wrapperDivSize = useElementSize(wrapperDivRef);
     const ensembleSet = useEnsembleSet(workbenchSession);
     const [plotType, setPlotType] = moduleContext.useStoreState("plotType");
+    const referenceSensitivityName = moduleContext.useStoreValue("referenceSensitivityName");
     const [channelEnsemble, setChannelEnsemble] = React.useState<Ensemble | null>(null);
     const [channelResponseData, setChannelResponseData] = React.useState<EnsembleScalarResponse | null>(null);
-
+    const [availableSensitivityNames, setAvailableSensitivityNames] = moduleContext.useStoreState("sensitivityNames");
     const responseChannel = moduleContext.useInputChannel("response", initialSettings);
 
     const [showLabels, setShowLabels] = React.useState(true);
@@ -91,6 +94,15 @@ export const view = ({ moduleContext, workbenchSession, workbenchSettings, initi
     }, [responseChannel, ensembleSet]);
 
     const sensitivities = channelEnsemble?.getSensitivities();
+    React.useEffect(
+        function propogateSensitivityNamesToSettings() {
+            const sensitivityNames = sensitivities?.getSensitivityNames().sort() ?? [];
+            if (!isEqual(sensitivityNames, availableSensitivityNames)) {
+                setAvailableSensitivityNames(sensitivityNames);
+            }
+        },
+        [sensitivities]
+    );
     const colorSet = workbenchSettings.useColorSet();
     const sensitivitiesColorMap = createSensitivityColorMap(
         sensitivities?.getSensitivityNames().sort() ?? [],
@@ -98,10 +110,15 @@ export const view = ({ moduleContext, workbenchSession, workbenchSettings, initi
     );
 
     let computedSensitivityResponseDataset: SensitivityResponseDataset | null = null;
-    if (sensitivities && channelResponseData) {
+    if (referenceSensitivityName && sensitivities && channelResponseData) {
         // How to handle errors?
+
         try {
-            const sensitivityResponseCalculator = new SensitivityResponseCalculator(sensitivities, channelResponseData);
+            const sensitivityResponseCalculator = new SensitivityResponseCalculator(
+                sensitivities,
+                channelResponseData,
+                referenceSensitivityName
+            );
             computedSensitivityResponseDataset = sensitivityResponseCalculator.computeSensitivitiesForResponse();
         } catch (e) {
             console.warn(e);

